config.yml

```diff
@@ -43,7 +43,19 @@ common_settings:
       key: A_VERY_COMMONLY_USED_PASSWORD
       value: "ThisIsAVerySecretPassword"
       variable_type: env_var #or file
-      protected: false #
+      protected: "false" #
+
+  group_delete_secret_variables:
```

Comment:
I suggest following the convention for deleting entities that we have for services (https://github.com/egnyte/gitlabform/blob/master/config.yml#L180) and hooks (https://github.com/egnyte/gitlabform/blob/master/config.yml#L194) - using the same config sections, but adding an extra dict element: `delete: true`